server/taskflows/hpccloud/taskflow/nwchem/neb.py

```diff
@@ -0,0 +1,54 @@
+import cumulus.taskflow.cluster
+from cumulus.taskflow.cluster import create_girder_client
+from cumulus.tasks.job import submit_job, _monitor_jobs
+from cumulus.tasks.job import download_job_input_folders
+from cumulus.tasks.job import upload_job_output_to_folder, job_directory
+from cumulus.transport import get_connection
+from . import setup_input, create_job, submit, submit_nwchem_job, \
+    monitor_nwchem_job, upload_output
+
+
+from hpccloud.taskflow.utility import *
+
+import sys
+
+class NWChemTaskFlow(cumulus.taskflow.cluster.ClusterProvisioningTaskFlow):
```

Comment:
We now have ```hpccloud.taskflow.nwchem.NWChemTaskFlow``` and ```hpccloud.taskflow.nwchem.neb.NWChemTaskFlow``` I can update the class name to make this more explicit if you think that is a good idea?